lib/cli/error.rs

```diff
@@ -8,7 +8,7 @@ use casper_types::{
     account::AccountHash, Key, NamedArg, PublicKey, RuntimeArgs, TimeDiff, Timestamp, URef,
 };
 use casper_types::{CLValueError, KeyFromStrError, UIntParseError, URefFromStrError};
-use uint::FromDecStrErr;
+pub use uint::FromDecStrErr;
```

Comment:
Can we leave this private and remove the reference to `FromDecStrErr` from `cli.rs`? We should be good, no?

Reply:
I expose this error to the SDK like CliError, I would prefer to keep it please because right now I did not really clean on that side in the SDK, it has to import those errors right now.

```
use casper_client::{
    cli::JsonArgsError,
    cli::{CliError, FromDecStrErr},
    Error as CasperClientError,
};
```

Reply:
fair enough, thanks